compact/materials.xml

```diff
@@ -399,4 +399,24 @@
     <fraction n="0.96" ref="W"/>
     <fraction n="0.04" ref="Polystyrene"/>
   </material>
+  <material name="SciFiPb_Scintillator">
+    <D type="density" value="1.049" unit="g / cm3"/>
+    <fraction n="0.077" ref="H"/>
+  	<fraction n="0.921" ref="C"/>
+   	<fraction n="0.001" ref="N"/>
+   	<fraction n="0.001" ref="O"/>
+    <constant name="BirksConstant" value="0.126*mm/MeV"/>
+  </material>
+  <material name="SciFiPb_Glue">
+    <D type="density" value="1.180" unit="g / cm3"/>
+    <fraction n="0.091" ref="H"/>
+  	<fraction n="0.822" ref="C"/>
+   	<fraction n="0.032" ref="N"/>
+   	<fraction n="0.055" ref="O"/>   
+  </material>
+  <material name="SciFiPb_PbGlue">
+    <D type="density" value="8.558" unit="g / cm3"/>
+    <fraction n="0.9622" ref="Pb"/>
+  	<fraction n="0.0378" ref="SciFiPb_Glue"/>
+  </material>
```

Comment:
Removed tabs.
```suggestion
  <material name="SciFiPb_Scintillator">
    <D type="density" value="1.049" unit="g / cm3"/>
    <fraction n="0.077" ref="H"/>
    <fraction n="0.921" ref="C"/>
    <fraction n="0.001" ref="N"/>
    <fraction n="0.001" ref="O"/>
    <constant name="BirksConstant" value="0.126*mm/MeV"/>
  </material>
  <material name="SciFiPb_Glue">
    <D type="density" value="1.180" unit="g / cm3"/>
    <fraction n="0.091" ref="H"/>
    <fraction n="0.822" ref="C"/>
    <fraction n="0.032" ref="N"/>
    <fraction n="0.055" ref="O"/>   
  </material>
  <material name="SciFiPb_PbGlue">
    <D type="density" value="8.558" unit="g / cm3"/>
    <fraction n="0.9622" ref="Pb"/>
    <fraction n="0.0378" ref="SciFiPb_Glue"/>
  </material>
```

Reply:
Sorry, this had to be some weird miscommunication with VSC. Should be fixed now.